code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -166,7 +166,12 @@ GLOBAL_DATUM_INIT(multispin_words, /regex, regex("like a record baby"))
 				var/mob/living/carbon/human/H = L
 				if(H.check_ear_prot() >= HEARING_PROTECTION_TOTAL)
 					continue
-			listeners += L
+			if(istype(L, /mob/camera/aiEye))
+				var/mob/camera/aiEye/AEAIO = L
```

Comment:
this is funny, but also confusing. Can you change this to something like `ai_eye`

Reply:
will do osrry forgot